models/action.go

```diff
@@ -360,7 +360,7 @@ func UpdateIssuesCommit(doer *User, repo *Repository, commits []*PushCommit) err
 
 			issue, err := GetIssueByRef(ref)
 			if err != nil {
-				if IsErrIssueNotExist(err) {
+				if IsErrIssueNotExist(err) || err == errMissingIssueNumber {
 					continue
```

Comment:
`GetIssueByRef` will return many kinds of error besides `ErrIssueNotExist` and `errMissingIssueNumber`. Don't know how to fix this.